### PR TITLE
Remove declaration and update confirmation page

### PR DESCRIPTION
**Pull request by LucasCairns**
This PR includes;
- Removal of the declaration page
- Removal of the declaration from the PDF template
- Updated the confirmation page content
- Updates to a bunch of util functions and tests

PDF preview available when all tasks are complete
Screenshot 2022-06-10 at 13 38 56

PDF preview unavailable when all tasks are NOT complete
Screenshot 2022-06-10 at 13 38 41

Updated confirmation page
Screenshot 2022-06-10 at 13 39 28

### CODE REVIEW

File: app/upw/controllers/saveAndContinue.js [hunk 1]
```diff
@@ -72,8 +55,7 @@ class SaveAndContinue extends BaseSaveAndContinue {
 
   saveValues(req, res, next) {
     const answers = req.sessionModel.get('formAnswers') || {}
-    const answersWithInvalidatedDeclarations = invalidateDeclarations(answers)
-    req.sessionModel.set('answers', answersWithInvalidatedDeclarations)
+    req.sessionModel.set('answers', answers)
```
Comment (Coupey) [inline app/upw/controllers/saveAndContinue.js:58]: Just need to double check that this is still the right thing to do, after the changes we made yesterday.

**Review by Coupey — APPROVED** [review]
Nice!

File: app/upw/controllers/saveAndContinue.js (hunk 1 above)
Comment (LucasCairns) [reply app/upw/controllers/saveAndContinue.js:58]: Good point! will rebase against `main` and double check

Comment (LucasCairns) [reply app/upw/controllers/saveAndContinue.js:58]: Rebased and this is looking okay in the diff

**Pull request merged**